ledger/error/src/main/scala/com/daml/error/definitions/LedgerApiErrors.scala

```diff
@@ -825,6 +739,80 @@ object LedgerApiErrors extends LedgerApiErrorGroup {
       ) extends LoggingTransactionErrorImpl(cause = cause)
     }
 
+    @Explanation(
+      "The ledger time of the submission violated some constraint on the ledger time."
+    )
+    @Resolution("Retry the transaction submission.")
+    object InvalidLedgerTime
+        extends ErrorCode(
+          id = "INVALID_LEDGER_TIME",
+          ErrorCategory.InvalidGivenCurrentSystemStateOther, // It may succeed at a later time
+        ) {
+      case class RejectEnriched(
+          details: String,
+          ledger_time: Instant,
```

Comment:
Minor: use camelCase?

Reply:
Actually, we can't. See previous discussion on the topic https://github.com/digital-asset/daml/pull/11622#discussion_r748033530